utils/cli/cli.c

```diff
@@ -0,0 +1,657 @@
+/*
+ * BlueALSA - cli.c
+ * Copyright (c) 2016-2021 Arkadiusz Bokowy
+ *
+ * This file is a part of bluez-alsa.
+ *
+ * This project is licensed under the terms of the MIT license.
+ *
+ */
+
+
+#if HAVE_CONFIG_H
+# include <config.h>
+#endif
+
+#include <errno.h>
+#include <getopt.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <unistd.h>
+
+#include "shared/dbus-client.h"
+#include "shared/defs.h"
+#include "shared/log.h"
+
+
+/**
+ * Helper macro for internal usage. */
+# define print_error(M, ...) if (!quiet) { error(M, ##__VA_ARGS__); }
```

Comment:
I would add another macro like this:
```c
#define cmd_print_error(M, ...) if (!quiet) { error("%s: " M, argv[0], ##__VA_ARGS__); }
```

and use it inside `cmd_*` functions. 

Right now we've got:
```
bluealsa-cli info /org/bluealsa/hci0/dev_1C_48_F9_9D_81_5C/hfpag/source foo
bluealsa-cli: E: Too many arguments
```

After this change it would be:
```
bluealsa-cli info /org/bluealsa/hci0/dev_1C_48_F9_9D_81_5C/hfpag/source foo
bluealsa-cli: E: info: Too many arguments
```
or maybe format it like this:
```
bluealsa-cli: E: CMD "info": Too many arguments
```

What do you think?